tests/integration/05.Dfw.Tests.ps1

```diff
@@ -1942,12 +1942,17 @@ Describe "DFW" {
             $rule.action | should be deny
             $rule.disabled | should be "false"
             $rule.logged | should be "true"
-            $rule = Get-NsxFirewallSection -Name $l3sectionname | Get-NsxFirewallRule -Name "pester_dfw_rule1" | Set-NsxFirewallRule -name "modified_pester_dfw_rule1" -action allow -disabled:$true -logged:$false
+            #There is no comment before, it will be add
+            $rule = Get-NsxFirewallSection -Name $l3sectionname | Get-NsxFirewallRule -Name "pester_dfw_rule1" | Set-NsxFirewallRule -name "modified_pester_dfw_rule1" -action allow -disabled:$true -logged:$false -comment "My Comment"
             $rule | should not be $null
             $rule.name | should be "modified_pester_dfw_rule1"
             $rule.action | should be allow
             $rule.disabled | should be "true"
             $rule.logged | should be "false"
+            $rule.notes | should be "My Comment"
+            #There is already a comment, it will be erase
```

Comment:
Please change the wording to `"it will be replaced"` as I got confused as to why there was still a value when you said it would be erased.